client/parser/parser.go

```diff
@@ -966,7 +966,7 @@ func (p *parser) voiceGroup() (ASTNode, error) {
 
 	voiceGroupNode := ASTNode{
 		Type:          VoiceGroupNode,
-		SourceContext: firstVoiceMarkerToken.sourceContext,
```

Comment:
@daveyarwood small bug where we didn't suppress for this token.

Reply:
Nice catch!